types/serialization.go

```diff
@@ -183,7 +181,8 @@ func (d *Data) ToProto() *pb.Data {
 	return &pb.Data{
 		Txs:                    txsToByteSlices(d.Txs),
 		IntermediateStateRoots: d.IntermediateStateRoots.RawRootsList,
-		Evidence:               evidenceToProto(d.Evidence),
+		// Note: Temporarily remove Evidence #896
+		// Evidence:               evidenceToProto(d.Evidence),
```

Comment:
This should be tested using actual cosmod-sdk app to ensure that cosmos-sdk is not panicing or failing validation somewhere.

Reply:
tested gm tutorial with a sequencer and a full node in local-devnet and it does not panic. so we are good to go.